doc/charm++/manual.rst

```diff
@@ -8711,6 +8711,38 @@ computational power because those dedicated cores are not utilized at
 all during most of the application's execution time. This case indicates
 the necessity of a unified runtime supporting both types of parallelism.
 
+
+Tass with StealQ for within-node load balancing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Converse, the underlying scheduling-and-communication system for Charm++, supports a steal queue as abuild time option,
```

Comment:
Typo: "a build". Also I thinking it should be described first as a working-stealing task queue, potentially abbreviated as steal queue later.